### ISSUE TITLE
Running container as different user fails with permission denied error

### ISSUE THREAD

**Issue by irfadrazick** [coderaiser/cloudcmd#182]
* **Version**: `coderaiser/cloudcmd:10.4.1`
* **Node Version**: `v10.6.0`
* **OS**: `Linux infosak 4.4.0-127-generic #153-Ubuntu SMP Sat May 19 10:58:46 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux`
* **Browser name/version**: `Chrome/67`
* **Used Command Line Parameters**: `docker run --name cloudcmd -u 10000 -p 8000:8000 -v /var/www:/var/www -v /var/log:/var/log coderaiser/cloudcmd:10.4.1` 
* **Changed Config**: Using different user to run container

When I use different user other than default root to launch a container I am unable to get it started as it tries to switch to `/root` directory and gets permission denied.
```
docker: Error response from daemon: OCI runtime create failed: container_linux.go:348: starting container process caused "chdir to cwd (\"/root\") set in config.json failed: permission denied": unknown.
```
I have a use case where I allow access to file which has sensitive information only to web user `www-data` where other users cannot view that file, hence it makes sense to run the container as different user. 

In the Dockerfile I see WORKDIR is changed to `/root`. Removing the line should fix this problem.

https://github.com/coderaiser/cloudcmd/blob/4c453e22be1b8860fc8f43caf432d978cdd3e936/Dockerfile#L16

Please kindly check and do the needful.

Thank you

**Comment by coderaiser** [comment]
That is very strange. I can not reproduce this. When I run

```
docker run --name cloudcmd -u 10000 -p 8000:8000 -v /var/www:/var/www -v /var/log:/var/log coderaiser/cloudcmd:10.4.1
```

Everything works good with `docker v17`.
What version of `docker` do you use?

**Comment by coderaiser** [comment]
Closed due to a long time of inactivity.

**Comment by irfadrazick** [comment]
Sorry for the delay, my docker version is `18.06.0-ce, build 0ffa825`

**Comment by aaron-sua** [comment]
I am seeing this issue as well.
I typically start a container with a service user, let's say 'appuser'
if I then try to docker exec -it -u root
I get the above error
If I change the working_dir to something root has direct access, like /tmp
it works fine

**Comment by luisbrandao** [comment]
Me too, in many conteiners. ismaleiva90/weblogic12 and the oficial jenkins one are exemples.
I cant get root in those conteiners

**Comment by coderaiser** [comment]
The thing is `root` is default user in docker images used by `Cloud Commander`, and `WORKDIR` was set to `root` as a home directory of `root` user, as [shown in example](http://cloudcmd.io/#docker):

```sh
docker run -t --rm -v ~:/root -v /:/mnt/fs -p 8000:8000 coderaiser/cloudcmd
```

To have ability to see home directory files straight after run.

**Comment by coderaiser** [comment]
Fixed with d16e345 🔨. Landed in [v11.8.2](https://github.com/coderaiser/cloudcmd/releases/tag/v11.8.2) 🎉.

Is it works for you?
